jaws-journey-deploy/orb.yml

```diff
@@ -627,6 +627,7 @@ jobs:
     resource_class: << parameters.resourceClass >>
     machine:
       docker_layer_caching: false
+      image: ubuntu-2004:202201-02
```

Comment:
what is the purpose of this change?

Reply:
would it not be better to pass this in rather than hardcoding? i.e. << parameters.image >>

Reply:
yeh probably makes sense to just specify it once. I'll try something like << parameters.image >>

Reply:
@ChrisFord-Ovo pushed up some changes to stick this in a parameter

Reply:
cool - yeah also means consumers of the orb could change the image if they wanted - although not sure why